src/serializer/ResidualHeapInspector.js

```diff
@@ -156,6 +156,7 @@ export class ResidualHeapInspector {
           desc.value !== undefined &&
           !this.realm.isCompatibleWith(this.realm.MOBILE_JSC_VERSION) &&
           !this.realm.isCompatibleWith("mobile") &&
+          desc.writable === true &&
```

Comment:
I don't think `desc.writable === true` should be added to the conjunction like this.
According to this
https://github.com/facebook/prepack/blob/ea0624b10863f5981c9fb8d5d03e64fa0d798ca1/src/methods/function.js#L738
Then we cannot ignore the current property descriptor if
`enumerable !== false || writable !== false || configurable !== true`, but that should be `||` to the whole condition, not `&&`.

Reply:
Basically, add another statement
```js
if (desc.enumerable !== false || desc.writable !== false || desc.configurable !== true) return false;
```

Reply:
Like in my latest commit?